File: smartPlantSpringBoot/_/src/main/java/com/app/tddt4iots/entities/Notification.java
package com.app.tddt4iots.entities;

import com.app.tddt4iots.enums.*;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.util.*;
import java.lang.*;

import jakarta.persistence.*;

@Entity
@Table(name = "Notification")
@Data
@NoArgsConstructor
public class Notification {
    
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;
     
    @Column(name = "element", nullable = true, unique = true, length = 30) 
    private String element; 
 
    @Column(name = "message", nullable = false, unique = false, length = 30) 
    private String message; 
 
    
}    

File: smartPlantAngular/src/app/Models/Notification.ts
export class Notification {
  get message(): string {
    return this._message;
  }

  set message(value: string) {
    this._message = value;
  }
  get element(): string {
    return this._element;
  }

  set element(value: string) {
    this._element = value;
  }
  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }
  private _id!: number;
  private _element!: string;
  private _message!: string;
}
